src/test/resources/alma/(DE-605)HT000312236.json

```diff
@@ -180,7 +180,7 @@
     },
     "resultOf" : {
       "type" : [ "CreateAction" ],
-      "endTime" : "2021-05-21T11:47:53",
+      "endTime" : "2021-06-01T16:48:11",
```

Comment:
we agreed to not commit test files which only changes in "endTime"

Reply:
I know since this is a Work in Progress and a PR-draft this was not about commit a final result but illustrating the the broader scope of the change.
Adrian and I decided on Wednesday that we can go on from here an adapte the default change from the ALMA transformation.
I will set up a new PR with clean changes.